tests/benchmarks/test_result.py

```diff
@@ -20,12 +22,24 @@ def test_add_raw_data():
     result = BenchmarkResult('model', BenchmarkType.MODEL, ReturnCode.SUCCESS)
     result.add_raw_data('metric1', [1, 2, 3])
     result.add_raw_data('metric1', [4, 5, 6])
-
     assert (result.raw_data['metric1'][0] == [1, 2, 3])
     assert (result.raw_data['metric1'][1] == [4, 5, 6])
     assert (result.type == BenchmarkType.MODEL)
     assert (result.return_code == ReturnCode.SUCCESS)
 
+    assert (int(os.getenv('EXPORT_RAW_DATA', '1')) == 1)
```

Comment:
Why we test the default value?

Reply:
If the default value is not as expected, other test cases (check the benchmark results) may also fail. And one check here to directly expose it.

Reply:
Therefore, we should save the environment setting first and set it to our expected value. At the end, we will resume the environment setting.

Reply:
I mean that I want to make sure the default value is '1', so I add one directly check here.

Reply:
removed.